rollup.config.js

```diff
@@ -8,7 +8,14 @@ export default [
       name: 'KeyCode',
       format: 'cjs',
       file: 'dist/keycode.cjs.js'
-    }
+    },
+    plugins: [
+      typescript({
+        tsconfigOverride: {
+          compilerOptions: { declaration: true }
```

Comment:
Why not have this in `tsconfig.json`? We can instead disable it for `test.ts`.

Reply:
Done. Now that all the `*.d.ts` files for tests go inside gitignored directory `test/dist` it's not a problem.